azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebContainer.java

```diff
@@ -68,7 +69,7 @@ public static WebContainer fromString(String input) {
         if (StringUtils.isEmpty(input)) {
             return WebContainer.JAVA_OFF;
         }
-        if (StringUtils.equalsAnyIgnoreCase(input, JAVA, JAVA_8, JAVA_11)) {
+        if (StringUtils.equalsAnyIgnoreCase(input, JAVA, JAVA_7, JAVA_8, JAVA_11, STRING_JAVA_SE)) {
```

Comment:
Shall we think if the input starts with 'java'(ignore case), it is 'java se'?

Reply:
users may have some other invalid values like `javascript`, so just accept fixed container value as java se